contracts/deployables/erc20/VotesERC20LockableV1.sol

```diff
@@ -18,13 +18,14 @@ contract VotesERC20LockableV1 is IVotesERC20LockableV1, VotesERC20V1 {
         _disableInitializers();
     }
 
-    modifier isTransferable(address from) {
+    modifier isTransferable(address from, address to) {
         if (
             _locked &&
             // overrides while locked
             !(from == owner() || // owner can always transfer
                 _whitelisted[from] || // whitelisted addresses can always transfer
-                from == address(0)) // can always mint when locked
+                from == address(0)) && // can always mint when locked
+            to != address(0) // can always burn when locked
```

Comment:
With the previous suggestion, I think this condition will need to be looked at again...

We probably want to only allow whitelisted accounts to burn when locked.

Reply:
That's my understanding, yes.